components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1565,7 +1591,9 @@ def getControllingCondition(interfaceMember, descriptor):
             PropertyDefiner.getStringAttr(interfaceMember,
                                           "Pref"),
             PropertyDefiner.getStringAttr(interfaceMember,
-                                          "Func"))
+                                          "Func"),
+            PropertyDefiner.getStringAttr(interfaceMember,
```

Comment:
I _suspect_ that we will want a different method than getStringAttr because Exposed can contain multiple values?